stack-easybib/recipes/deploy-easybib.rb

```diff
@@ -0,0 +1,40 @@
+node['deploy'].each do |application, deploy|
+  case application
+  when 'easybib'
+    next unless allow_deploy(application, 'easybib', 'nginxphpapp')
+
+  when 'easybib_api'
```

Comment:
This can be removed, it is now everywhere  `stack-citationapi::role-publicapi` used that api